stdlib/docs/development.md

```diff
@@ -263,7 +263,7 @@ ls **/*.mojo | entr sh -c "./scripts/build-stdlib.sh && mojo main.mojo"
 Now, every time you save a Mojo file, it packages the standard library and
 runs `main.mojo`.
 
-**Note**: you should stop `entr` while doing commits, otherwise you could have
+__Note__: you should stop `entr` while doing commits, otherwise you could have
```

Comment:
**Question** Is this change because of differences in the `markdownlint-cli` version we use internally, or something else?

Reply:
That may be possible I can try it out. Give me a sec

Reply:
I tried with 0.34.0 and I still got 

```
stdlib/docs/development.md:266:1 MD050/strong-style Strong style should be consistent [Expected: underscore; Actual: asterisk]
```
Looking at the documentation it makes sense: https://github.com/DavidAnson/markdownlint/blob/main/doc/md050.md

By default markdownlint just wants all emphasis to be the same. Being `__something__` or `**something**`. I don't know what the tool installed on your machine is doing but I would say that the pre-commit is right here.

If you want we can add some configuration to enforce one style or the other, that would avoid any blurry line concerning this rule. What do you think?

Reply:
I see, that makes sense.  I don't feel strongly on the version, but I'd just like our internal and external to be consistent with the style they pick to avoid drift.  @modularml/docs do you have opinions here?